github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/dsl/context/GitHubPRTriggerDslContext.java

```diff
@@ -33,6 +35,14 @@ public void setPreStatus() {
         setPreStatus = true;
     }
 
+    public void setCancelQueued() {
```

Comment:
I was just following the style that was there. So you suggest to change even the existing ones?